server/model/organization.js

```diff
@@ -14,6 +15,7 @@ export const wellcomeCollection: Organization = {
   name: 'Wellcome Collection',
   url: 'https://wellcomecollection.org',
   logo: 'https://wellcomecollection.org/client/icons/apple-touch-icon.png',
+  twitterHandle: '@ExploreWellcome',
```

Comment:
It'd be good to figure out how we infer `sameAs` from `twitterHandle`, but, personally, I've no idea yet...